### DISCUSSION TITLE
Crater template map

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1398] [Maps and Content Showcase]
So, i've created a template map set inside a big crater, which itself is a big model.
There is a very small platform you spawn on, in the middle, which you can start to build your things on, as you can't spawn things in the top of a model itself.
Water is there more for aesthetic purposes, since there is no death material inside it, so you can just swim away if you like.
The crater model itself has collisions, so you can walk on it without falling down to your death, or even use it as part of the level design itself.
Crater was created from scratch using World Creator 1, texture in 2k, is a mix of the one created using the shaders of the program and gimp, plus a edited in ambient occlusion i baked from the high res mesh, along with the 4k normal map baked on blender (the original normal map from the program didn't quite work, so i had to re-bake it).

![20230105_111324](https://user-images.githubusercontent.com/24359960/210804570-5690b972-f4c3-43bd-85a9-29d4eb2ac597.png)
![20230105_110640](https://user-images.githubusercontent.com/24359960/210805111-62922745-c573-4186-b647-96f964857d80.png)
![20230105_110647](https://user-images.githubusercontent.com/24359960/210805165-3e6ff7ae-e999-4a19-b62b-27afac016c60.png)
![20230105_110719](https://user-images.githubusercontent.com/24359960/210805203-c1fbfb74-5599-4a4d-b3b7-a0d0f7d1626f.png)

Night variant:
![20230105_111802](https://user-images.githubusercontent.com/24359960/210805429-e2c2ab2f-6d97-44f0-bd4d-b4fdc06e96e2.png)
![20230105_111814](https://user-images.githubusercontent.com/24359960/210805435-9f926076-0645-4449-b976-e7b6121df962.png)
![20230105_111837](https://user-images.githubusercontent.com/24359960/210805437-04b5308e-6eb0-40ea-8629-0d750559cc51.png)

Top view, without water:

![20230105_110759](https://user-images.githubusercontent.com/24359960/210805513-aaf201d8-7c38-4cba-b4f1-33ed54ccac63.png)

File:
[crater-template-map.zip](https://github.com/redeclipse/base/files/10352966/crater-template-map.zip)

Remember, this is just a template to build something upon, so  have fun!

**Comment by dc-redeclipse** [comment]
awsome! It screms on me to make a small map with it.
